doc/digraph.xml

```diff
@@ -1083,3 +1083,145 @@ gap> JohnsonDigraph(1,0);
   </Description>
 </ManSection>
 <#/GAPDoc>
+
+<#GAPDoc Label="DigraphNumber">
+<ManSection>
+  <Oper Name="DigraphNumber" Arg="nr[, v]"/>
+  <Returns>A digraph.</Returns>
+  <Description>
+    This operation defines a bijection between the positive integers and the set
+    of all digraphs without multiple edges.  If <A>nr</A> is a positive integer,
```

Comment:
Is the word "finite" missing here? Otherwise, there is no such bijection, and the target is not even a set.